magda-web-server/src/index.ts

```diff
@@ -16,8 +20,22 @@ const argv = yargs
         type: "boolean",
         default: false
     })
+    .option("baseExternalUrl", {
+        describe:
+            "The absolute base URL of the Magda site, when accessed externally. Used for building sitemap URLs which must be absolute.",
+        type: "string",
+        default: "http://localhost:6100/",
+        required: true
+    })
+    .option("registryApiBaseUrlInternal", {
+        describe: "The url of the registry api for use within the cluster",
+        type: "string",
+        default: "http://localhost:6101/",
```

Comment:
this should be `http://localhost:6101/v0/` right?  I had to change it to that to make it work outside the cluster, at least.

Reply:
Oops, sorry!